DSNP-Wallet/Source/Common/Services/ServiceCoordinator.swift

```diff
@@ -41,12 +41,12 @@ extension ServiceCoordinator: ServiceCoordinatorProtocol {
         setup(chainRegistry: chainRegistry)
 
         accountInfoService.setup()
-        assetsService.setup()
+//        assetsService.setup()
```

Comment:
if `accountInfoService` is doing the setup and the throttling do we still need the `assetsService`?